ivy/functional/frontends/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 versions = {
     "torch": "2.0.1",
+    "torchvision": "0.15.2.",
     "tensorflow": "2.13.0",
     "numpy": "1.25.2",
     "jax": "0.4.14",
```

Comment:
The fixture error gets resolved by adding `conftest.py` under `test_torchvision`.
Should contain this simple code like the rest of the frontend test setups:
```
import pytest


@pytest.fixture(scope="session")
def frontend():
    return "torchvision"
```